tests/test_frameSet.py

```diff
@@ -83,21 +83,28 @@ def test_FrameSetPermutation(self):
 
         If one permutes the axes of the current frame of a frame set
         *in situ* (by calling `permAxes` on the frame set itself)
-        then mappings connected to that frame are also permuted
+        then mappings connected to that frame are also permuted.
+
+        Test a FrameSet with a different number of input and output axes,
+        to test DM-9899 FrameSet.permAxes will fail if nIn != nOut
         """
-        frame1 = astshim.Frame(2)
-        unitMap = astshim.UnitMap(2)
+        frame1 = astshim.Frame(1)
```

Comment:
There would be no harm to doing both. I'll see how difficult it is to put it back.